src/app/learn/accessibility/code/index.html

```diff
@@ -1,5 +1,5 @@
 <stache
-  pageTitle="Coding for accessibility"
+  pageTitle="Coding for Accessibility"
```

Comment:
We use a different casing that for page headings?

Reply:
Yep. The style is headline-style capitalization for the main header on the page, and sentence-style capitalization on all other headers.